config/jobs/kubernetes/sig-cloud-provider/azure/release-1.26.yaml

```diff
@@ -34,7 +34,7 @@ presubmits:
             - bash
             - -c
             - >-
-              cd ${GOPATH}/src/sigs.k8s.io/azuredisk-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&
+              cd ${GOPATH}/src/sigs.k8s.io/azuredisk-csi-driver &&
```

Comment:
This doesn't look right. 
Shouldn't this be `cd ${GOPATH}/src/sigs.k8s.io/azuredisk-csi-driver && ./deploy/install-driver.sh master local,snapshot,enable-avset &&` ?
I ran `./generate 1.27` as mentioned in the README.